pyproject.toml

```diff
@@ -85,8 +85,8 @@ tests = [
     # jsonargparse
     "jsonargparse[signatures]>=4.28.0",
     ## Style
-    # ruff 0.2+ required for [ruff.lint]
-    "ruff>=0.2",
+    # ruff 0.9+ required for 2025 style guide
```

Comment:
If you run with ruff 0.8, it undoes these changes. Let's require 0.9+ to make sure it stays the same.